### PR TITLE
Adds new chaplain armor to the armament beacon

### PR DESCRIPTION
**Pull request by NoxVS**
### About The Pull Request
Adds a new set of chaplain armor to the armament beacon using admin only crusader armor that I believe was used for the Hand of God mode
Screenshot (818)
### Why It's Good For The Game
Adds more variation to chaplain aesthetics, I felt the previous armament beacon armors weren't varied enough like the null rod options are
### Changelog
:cl: NoxVS
rscadd: New chaplain armor using unused crusader armor
/:cl:

### CODE REVIEW

**Comment by NoxVS** [comment]
First PR so I am sure I fucked something up horribly somehow

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -124,6 +124,25 @@
...
+/obj/item/storage/box/holy/adept
+	name = "Divine Adept Kit"
+
+/obj/item/storage/box/holy/adept/PopulateContents()
+	new /obj/item/clothing/suit/armor/riot/chaplain/adept(src)
+	new /obj/item/clothing/head/helmet/chaplain/adept(src)
+
+/obj/item/clothing/head/helmet/chaplain/adept
+	name = "adept hood"
+	desc = "Its only heretical when others do it"
```
Comment (Detective-Google) [inline code/game/objects/items/holy_weapons.dm]: small gripe but the other desc's for these have periods at the end

**Comment by Quartich** [comment]
just a note but for screenshots nearest neighbor and a higher icon size makes it look better

File: code/game/objects/items/holy_weapons.dm (hunk 1 above)
Comment (NoxVS) [reply code/game/objects/items/holy_weapons.dm]: Thanks, fixed it

**Comment by NoxVS** [comment]
It no longer covers the mouth

**Pull request merged**